backend/src/main/java/eventlistener/controller/EventController.java

```diff
@@ -37,4 +43,14 @@ public Event getSingleEvent(@PathVariable Long eventId){
     public Event addEvent(@RequestBody Event eventToAdd){
         return userEventService.addEvent(eventToAdd);
     }
+
+    @PostMapping("/trigger/{eventId}")
+    public void triggerEventAction(@PathVariable Long eventId, @RequestBody EventContentDTO eventDetails){
+        eventMailService.triggerEvent(eventId, eventDetails);
+    }
+
+    @PutMapping("/{eventId}")
+    public Event editEvent(@PathVariable Long eventId, @RequestBody EventToModifyDTO event){
```

Comment:
How is a EventToModifyDTO different from an EventDTO? I would use the same and different constructors in case there's some attributes lacking in one of the two.